File: AndroidStudioProjects/LSTEVEW/app/src/main/java/com/example/lstevew/DetayActivity.java
package com.example.lstevew;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.graphics.Bitmap;
import android.os.Bundle;

import android.widget.ImageView;
import android.widget.TextView;

import static com.example.lstevew.MainActivity.secilenResim;


public class DetayActivity extends AppCompatActivity
{

    ImageView resim;
    TextView acıklama;


    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detay);
        resim = findViewById(R.id.imageView);
        acıklama = findViewById(R.id.textView2);


        Intent al = getIntent();
        String a = al.getStringExtra("name");
        acıklama.setText(a);

        resim.setImageBitmap(secilenResim);
    }
   /*
    public void değiştir(){
        ImageView ımageView =  findViewById();
        Button button = findViewById();
        ımageView.setImageResource(R.drawable.Değiştirlecek olan esimİsmi);
    }
    */

}

File: AndroidStudioProjects/ListViewkullanm/app/src/main/java/com/example/listviewkullanm/DetayActivity.kt
package com.example.listviewkullanm

import android.graphics.BitmapFactory
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_detay.*

class DetayActivity : AppCompatActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detay)

        var al = intent
        var a = al.getStringExtra("name")
        textView2.text = "" + a
        val gelenResimId = al.getIntExtra("resim",0)

        val gelenResimBitmap = BitmapFactory.decodeResource(this.resources,gelenResimId)
        imageView.setImageBitmap(gelenResimBitmap)

    }
}